### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Signing in to Azure Storage_Toggle Explorer Pane - Tree View_Keyboard: Horizontal scroll bar available in Tree view section is not accessible using keyboard.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/AzureStorageExplorer#7579]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1

### Regression From

No

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

Windows

### OS Version

Windows 11

### Bug Description

**GitHub Tags:**
#A11yMAS;#A11yTCS;#Win11;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_Win32_Dec2023;#Azure Tools Storage Explorer;#FTP;#WCAG2.1.1;#Keyboard;#A11yCAIOH;#CEACCReviewed;

### Environment Details:

Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)

### Steps to Reproduce

1. Launch Storage Explorer
2. TAB to "Open Connect dialog" pane and Sign in to Azure.
3. TAB to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Traverse through the treeview opened on left of screen for whatever tasks or subscription selected.
5. Observe that when we trying to scroll the horizontal scroll bar available in Tree view section is not accessible using keyboard.

### Actual Experience

In order to view or access the data present is the tree view section using keyboard user is unable to scroll the horizontal scrollbar.

### Expected Experience

To view or access the complete data present in Tree view section user should be able to access the horizontal scroll bar via keyboard.

### User Impact:

Keyboard only users will not be able to view or access the data present in tree view section if scroll bars are not accessible using keyboard.

![keyboard scroll bar](https://github.com/microsoft/AzureStorageExplorer/assets/115216656/6a8aeb9e-1c3f-4a7d-a637-694695c92b72)

**Comment by MRayermannMSFT** [comment]
Horizontal scroll bar has been removed, overflowing text will now have an ellipsis ("...") at the end to signify overflow.

**Comment by Rama8345** [comment]
@MRayermannMSFT 

Here expected is that either scroll bar should be accessible via keyboard or tooltips should be provided for the truncated texts and that too accessible via keyboard. Removing scroll is not the exact solution because anyhow information is missing for keyboard only users. Please provide latest build with the fixes.

**Comment by MRayermannMSFT** [comment]
Hi @Rama8345 thanks for the feedback. I talked with the C+AI accessibility team and was told that as long as the content in the tree view which requires scrolling is visible elsewhere, then this issue can be closed. Therefore, in an upcoming build, the following change will be made available:

In the Properties panel below the tree, everyone node will have a "Display Name" property. This will have the same value as the text for the node. This ensures that the text for the node is always keyboard accessible, even if the tree view's content requires scrolling.
![7579](https://github.com/microsoft/AzureStorageExplorer/assets/21112697/4cde9d1b-7720-49dd-b248-eaae29e99e4f)

We will share the new build with you and your team in the near future. Thanks.

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
version: 1.33.0-dev(96)
Issue is fixed. hence closing the bug. Please find below added attachment for reference.
![7579_Fixed](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/12d14e96-cd33-41d7-a1ef-ae562eaae14f)
